internal/controller/ai_gateway_route.go

```diff
@@ -25,6 +25,11 @@ import (
 const (
 	managedByLabel        = "app.kubernetes.io/managed-by"
 	expProcConfigFileName = "extproc-config.yaml"
+
+	conditionReconciled           = "Reconciled"
```

Comment:
Please make sure everything has a proper prefix when it's defined in a global package level context 

```suggestion
	aiGatewayRouteConditionTypeReconciled           = "Reconciled"
```